_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation.vue

```diff
@@ -381,25 +406,36 @@ export default {
     },
   },
   computed: {
-    filtersChosen() {
-      return this.foundSsc?.productFilters
-      ?? this.$store.state.smartShoppingCampaigns.filtersChosen;
+    filtersChosen: {
+      get() {
+        return (
+          this.foundSsc?.productFilters
+          ?? this.$store.state.smartShoppingCampaigns.filtersChosen
+        );
+      },
+      set() {},
```

Comment:
If this is a read only computed, remove the `get() / set()` ?

Reply:
I have a strange warn if I remove it : 
`[Vue warn]: Computed property "filtersChosen" was assigned to but it has no setter.
`
Do you know how to do ?

Reply:
Seeing this warning is a good thing, I understand there's a side effect somewhere.
There's a place in the code changing the value of `filtersChosen` that should not happen. Finding it should allow us to find another solution.

Reply:
Indeed we had a side effect, filtersChosen was set in another function as well 👌